### DISCUSSION TITLE
デバッグルーム、コントロールルームの完成度

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#68] [エリア・建築の完成度・開発状況]
- [x] 旧から新のデバッグルーム移行
_現状1.16環境でないので、完全な機能の移行ができない。_
_また他の建築で忙しく、こっちの建築は二の次になっている。_
- [ ] 特殊床用の例ステージの作成
_1.16環境にならないとシステムが確定しない以上下手に増やしても…と悩んでいる。_
_特殊床で作ってはならない例題等を展示したりする目的もあるので、いつかは完成させないとなぁとは思っている。_
- [ ] ~~全アイテムを新コントロールルームへ移行~~
_function管理になった以上、最終的に現行の初期地点の虹内部は消滅するのでコントロールエリアに既存のアイテムなどを移動するようにしたい。_
_デバッグルーム管理でもいいが、下手にオリジナルアイテムをデバッグルームにおいてしまうと悪用される可能性があるので_
_コントロールルームにオリジナルアイテムを置くべきだと思ってる。_

**Comment by NePonpon** [comment]
#67 ディスカッションにてデバッグルームの使い方を記述していく
